src/components/Tabs_UI.js

```diff
@@ -0,0 +1,63 @@
+import React, {Component} from 'react';
+import './Tabs_UI.css';
+
+class Tabs_UI extends Component {
+
+    constructor(props) {
+            super(props);
+            this.state = { 
+                openTabName: 'tab1'
+            }
+    }
+
+    changeState = (newTabName) => {
+        this.setState({ 
+            openTabName: newTabName
+        })
+    }
+    
+    render () {
+
+            return (
+
+                <div className="tabsContainer">
+
+                    <div className="tab">
+                    {/* <button onClick = { () => this.updateState(id)} className="buttonStyle">Tab 1</button> */}
+                    <button onClick={ () => this.changeState('tab1') } className="buttonStyle">Tab 1</button>
+                    <button onClick={ () => this.changeState('tab2') } className="buttonStyle">Tab 2</button>
+                    <button onClick={ () => this.changeState('tab3') } className="buttonStyle">Tab 3</button> 
+                    </div>
+
+
+    
+                    <div className={`tabContent ${this.state.openTabName === 'tab1' ? "show" : "hide" }`}>
```

Comment:
incearca sa scoti la inceput de functie proprietatiile din this.state / this.props

ex. `const { name } = this.state` => si atunci in loc sa scrii de fiecare data `this.state.name` poti folosi doar `name`.

Daca ai mai multe proprietati, poti pur si simplu sa le pui in acelas destructuring la inceput de functie, ex. `const { name, color, width, height } = this.state`